### PR TITLE
Jb49791

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by vigejolla** [comment]
> I haven't seen those FILEPATHs, but as long as we don't change the existing STRINGs your changes LGTM

Actually, now I see them as well. Well, the main point is that now it works :)

**Comment by martyone** [comment]
Extended CMAKE_SYSROOT mapping to both PATH and STRING types, added QT_QMAKE_EXECUTABLE handling.

File: src/tools/merssh/command.cpp [hunk 1]
```diff
@@ -276,18 +276,35 @@ void Command::maybeDoCMakePathMapping()
         }
 
         QString data = QString::fromUtf8(reader.data());
+
```
Comment (vigejolla) [inline src/tools/merssh/command.cpp:279]: I believe whitespace changes on unrelated lines in a same commit with functional changes are against Qt commit policy. Just saying, even though I'm not sure the Qt commit policy is really relevant here.

Comment (martyone) [reply src/tools/merssh/command.cpp:279]: Roughly speaking, I believe it makes crucial difference whether it is a change to existing non-blank lines or not. The reason for splitting changes into smaller portions is to allow determining the origin/purpose of code, and vertical space is rarely interesting in this respect. So I am OK with changes like this :)
